.github/workflows/integration-tests.yaml

```diff
@@ -56,27 +56,27 @@ jobs:
         working-directory: example/
 
       - name: Pull docker images
-        run: docker-compose pull
```

Comment:
[https://github.com/orgs/community/discussions/116610#discussioncomment-8997411](https://github.com/orgs/community/discussions/116610#discussioncomment-8997411)

Might be a better solution for this, but this does the trick. Looks like this has been broken for a while now